src/DotNetProjectFile.Analyzers/MsBuild/Project.Property.cs

```diff
@@ -38,6 +38,14 @@ public bool IsDevelopmentDependency()
     [DebuggerBrowsable(DebuggerBrowsableState.Never)]
     private bool? managePackageVersionsCentrally;
 
+    public OutputType.Kind GetOutputType()
+        => Property<OutputType>()?.Value
+        ?? Sdk switch
+        {
+            "Microsoft.NET.Sdk.Web" => OutputType.Kind.Exe,
```

Comment:
Can you add a test case for this SDK case?

Reply:
It a bit more work (as I can not rely on an inlineproject due to the fact that I have to define an main entrypoint, but I'll do it)

Reply:
perhaps it's easier to alter the inlineproject setup to also always add an empty entrypoint, so that we can use it for other test cases in the future as well

Reply:
Might be an idea. I added a project for now, and that was also not too much of an hassle after all.